### PR TITLE
Added skip_if option

### LINKED ISSUE #17: Enhancement google bot
Hi, great plugin!

What do you think adding more options for search engine crawlers?

For example skipping redirect for bots. Or skipping redirect for certain user agents.
thank you

### PR DESCRIPTION
**Pull request by afdev82**
This will fix #17

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -127,10 +127,18 @@ your project:
...
+You can pass a block to the `:skip_if` option to provide custom logic to skip the redirection.
+
+    Rails.application.middleware.use GeoRedirect::Middleware, skip_if: ->(req) { req.bot? }
+
+will ignore requests from bots (the code inside the block is just pseudo-code).
+
 ### Debugging
 
 You can add a `logfile` path string when adding the middleware if you want it to
-log some of its decision process into the file.
```
Comment (n0nick) [inline README.md]: these trailing spaces are here for a purpose - they create newlines in the Markdown doc. please do not remove them

**Comment by n0nick** [comment]
looks good overall, let's
1. fix the lint error
2. add a test :)

File: README.md (hunk 1 above)
Comment (afdev82) [inline README.md]: You are right, unfortunately my text editor is setup to remove the trailing spaces...

Comment (n0nick) [inline README.md]: Yes, I'm familiar with such editor setups from some of my co-workers :)
If like them you're using Sublime, here's what you want: http://robandlauren.com/2013/11/21/configuring-sublime-text-markdown/

Comment (afdev82) [inline README.md]: Yes, exactly this. Thanks ;)

**Comment by n0nick** [comment]
Merged in f671d7ba

**Pull request closed without merging**